GUIBuilder.java

```diff
@@ -0,0 +1,76 @@
+package CleanNotePad;
+
+import java.awt.Dimension;
+import javax.swing.JFrame;
+import javax.swing.JMenu;
+import javax.swing.JMenuBar;
+import javax.swing.JMenuItem;
+import javax.swing.JScrollPane;
+import javax.swing.JTextPane;
+
+//This class creates the GUI for the simple notepad
+public class GUIBuilder extends JFrame{
+	LogicForPad LP=new LogicForPad();

```

Comment:
This might seem nit picky, and it may just be my preference, but I think the name LP for the LogicForPad object is too non-descriptive. I would have named it padLogic